src/main/java/com/microsoft/azure/relay/HybridConnectionClient.java

```diff
@@ -190,14 +200,17 @@ public String toString() {
 		headers.put(RelayConstants.SERVICEBUS_AUTHORIZATION_HEADER_NAME, Arrays.asList(token.join().getToken()));
 		HybridConnectionEndpointConfigurator configurator = new HybridConnectionEndpointConfigurator();
 		configurator.addHeaders(headers);
+		if (customHeaders != null) {
+			configurator.addHeaders(customHeaders);
+		}
 		ClientEndpointConfig config = ClientEndpointConfig.Builder.create().configurator(configurator).build();
 
 		try {
 			URI uri = HybridConnectionUtil.buildUri(this.address.getHost(), this.address.getPort(),
 					this.address.getPath(), this.address.getQuery(), HybridConnectionConstants.Actions.CONNECT,
 					trackingContext.getTrackingId());
 			WebSocketChannel channel = new WebSocketChannel(trackingContext, EXECUTOR);
-			return channel.getWebSocket().connectAsync(uri, this.operationTimeout, config).thenApply(result -> channel);
+			return channel.getWebSocket().connectAsync(uri, this.operationTimeout, config).thenApply($void -> channel);
```

Comment:
>$void [](start = 92, length = 5)

I was toying around when I came up with naming these throw-away parameters `$void`. Seems like it might have caught on?

Reply:
I liked it, seems clear that it's a null result

---
In reply to: [266164513](https://github.com/Azure/azure-relay-java/pull/35#discussion_r266164513) [](ancestors = 266164513)